src/typings.d.ts

```diff
@@ -0,0 +1,7 @@
+import "@web3-react";
```

Comment:
it defines window.ethereum so TypeScript won’t fail because of it, right? and inside that file we can define other (only global?) types?

Reply:
Yes, you can create declaration files like "something.d.ts" in the src folder, it will  add global types in the project. For example, types for global variables like window, ENV_*, Object.values, etc. 

Also you can extend or override libraries, for example 
```import {useTranslate} from 'i18n'``` could has types with translation keys from locale configs.

https://www.typescriptlang.org/docs/handbook/declaration-files/by-example.html